src/style.css

```diff
@@ -0,0 +1,146 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+
+  --light-color: #616070;
+  --color-ligthblue: #00acdc;
+}
+
+.body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.section {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.section__list {
+  margin: 0;
+}
+
+.product-cards {
+  padding: 0;
+}
+
+.product-cards__items {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  list-style: none;
+  padding: 0 16px 16px;
+
+}
+
+.image__item {
+  width: 160px;
+  height: 134px;
+  padding-top: 32px;
+  padding-left: 3px;
+}
+
+.list__items {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.product-cards__text {
+  margin: 0;
+  padding-top: 35px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product-cards__article {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--light-color);
+  padding-top: 4px;
+  margin: 0;
+}
+
+.product-cards__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+  line-height: 14px;
+  padding-top: 17px;
+}
+
+.rate__stars {
+  display: flex;
+  align-items: center;
+  width: 100px;
+  padding-left: 2px;
+}
+
+.stars__image {
+  height: 16px;
+  width: 16px;
+  padding-right: 4px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars__image--disactive {
```

Comment:
Number of positive stars will be always changed. You have to handle with it.                               I wrote (> :nth-child). but i don't know it's right or not. Nobody has reviewed my homwork
The rest of code is ok

Reply:
Thanks for review.